deeppavlov/models/torch_bert/torch_transformers_sequence_tagger.py

```diff
@@ -375,3 +386,32 @@ def load(self, fname=None):
                 self.epochs_done = checkpoint.get("epochs_done", 0)
             else:
                 log.info(f"Init from scratch. Load path {weights_path} does not exist.")
+            if self.use_crf:
+                weights_path_crf = Path(f"{self.load_path}_crf").resolve()
+                weights_path_crf = weights_path_crf.with_suffix(".pth.tar")
+                if weights_path_crf.exists():
```

Comment:
I think we should add warning in case if weights_path_crf doesn't exists, cause in the inference mode in case if, by some reason, crf weights wasn't downloaded or was deleted, crf object will be initialized with zeros and will give wrong answers. Am I right?
Also should we for training add handling two cases: training from scratch and fine-tuning from existing checkpoint?